client-templates/nexus2/.env.sample

```diff
@@ -0,0 +1,27 @@
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# The Base URL for your Nexus Repository Manager
+# If not using basic auth this will only be "https://<your.nexus.hostname>"
+BASE_NEXUS_URL=https://<username>:<password>@<your.nexus.hostname>
+
+# The URL to your Nexus Repository Manager
+NEXUS_URL=$BASE_NEXUS_URL/nexus/content
```

Comment:
realised it can work with just this, and then updated the [Nexus Integration document](https://docs.google.com/document/d/110yx59Qm-eN8rthJF1pZYlHbyp-MXVDMkQR3tnOUGGM/edit#) to tell users the correct language setup for groups and repositories

Reply:
yep makes sense